perf/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
How to run the perf test? `cmake . && make -j` does not produce any executables. I only have `xtm.test`.

Reply:
It is built only in release mode (because i think, that it is unmeaning to test performance in debug). Also you need to
build and install google benchmark on you computer (https://github.com/google/benchmark).

Reply:
It looks far from trivial, so I would describe it somewhere how to use it. However I don't really want to install anything, otherwise my system would clog with tons of garbage packets I installed just for testing something one time. It would be good to have all the non-standard deps inside of the repository. Could you add it as a git submodule? It uses CMakeLists, so it should not be a problem to integrate.

Reply:
I don't see any changes regarding this, so I won't run the perf tests, sorry. Don't want to install anything into my system.

Reply:
Fixed